lib/src/main/java/com/faskn/lib/ClickablePieChart.kt

```diff
@@ -72,11 +75,23 @@ class ClickablePieChart @JvmOverloads constructor(
             context.theme.obtainStyledAttributes(attrs, R.styleable.ClickablePieChart, 0, 0)
 
         try {
+            // Popup text
```

Comment:
@turkergoksu, Can you remove comment lines, because they are same with variable names.